mlir/test/Dialect/Vector/vector-transfer-permutation-lowering.mlir

```diff
@@ -31,6 +33,31 @@ func.func @xfer_write_transposing_permutation_map(
   return
 }
 
+// Even with out-of-bounds, it is safe to apply this pattern
+// CHECK-LABEL:   func.func @xfer_write_transposing_permutation_map_out_of_bounds
+// CHECK-SAME:       %[[ARG_0:.*]]: vector<4x8xi16>,
```

Comment:
NIT: Argument definitions are aligned with function body in the test below. Although `xfer_write_transposing_permutation_map` also follows this formatting. Both are valid I guess. 🤷

Reply:
Let's prioritise consistency - I will update this before landing. Thanks!